libsrc/_DEVELOPMENT/target/crt_program_exit.inc

```diff
@@ -37,10 +37,17 @@
             
                ; program jumps to caller
 
+              IF __CPU_INTEL__
```

Comment:
Is this not emulated by z80asm?

Reply:
I don't know. Not one I asked for. I'll check.

Reply:
As far as I can see `ld de,(**)` is not emulated for INTEL or GBZ80.
It throws a warning, and I can't otherwise find it.

@pauloscustodio Could it be added to z80asm, please?